script/Ciel/Day 14/14_01B_03_CIEL14B_4_1.txt

```diff
@@ -1,187 +1,226 @@
-"Were you always a creature
+"Were you always that kind of creature?
 }
 [sha:c11830ec83ee86eb0ad03ae5c9b1eda800528a7a]{
 -- Page 4, Offset 40255.
 -- 生きていられる生き物だった？”』
-that could live without needing to breathe?"
+One that can live without needing to breathe?"
 }
 [sha:bf5f8b6d28c3336329ec0206f6e78969e23f743f]{
 -- Page 4, Offset 40256.
 -- 数分後に待つ、俺の最期を告げていた。
-After a few minutes of waiting, she gives me the final warning.
+// Continuation of 40253.
+She informs me of the end that awaits me a few minutes from now.
 }
 [sha:1c20a2964f5d5cf1de2af9d7d005b9ed390292e5]{
 -- Page 5, Offset 40257. Mods: @n.
 -- 「ふっ
-"Kh
+"%{i}Kh%{/i}
 }
 [sha:3a9773f9daa810288f636f1e50a3bf4007127efc]{
 -- Page 5, Offset 40258. Glued.
 -- ―――、う、っ―――……！」
-―――, ugh―――!"
+――%{i}ugh%{/i}―――!"
 }
 [sha:1cf05b06e762c4739a66f435608ed583275781ed]{
 -- Page 5, Offset 40259.
 -- アルクェイドの言う通り、地上は動物が暮らせる環境ではなくなっていた。
-It's just as Arcueid says. The earth had turned into an environment that's unfit for animals to live.
+It's just as Arcueid says.
+ The earth is no longer an environment in which animals can survive.
 }
 [sha:c2fe5c4aceb974e9ff76e6822d1c21072876b3c0]{
 -- Page 5, Offset 40260.
 -- 40度を超える高温と、今も成長し続け、少しずつ隙間を埋めていく緑の舌。
-The temperature passes over 40 degrees, and even now, the green tongue continues to grow and slowly cover the remaining gaps.
+// 舌?? Assuming this is more like 'worm their way around'
+It's already over 40℃,
+ and the creeping fingers of greenery continue to grow and fill in the remaining gaps.
 }
 [sha:0bba3f350b7f0a190eec964481a4168cdced0047]{
 -- Page 6, Offset 40261.
 -- そしてなにより、満足に息ができない。
-Most of all, I'm unable to breathe properly.
+But most pressingly, I can't breathe properly.
 }
 [sha:2daea3b34e23f9b7e0565842c33cb5823a517f1b]{
 -- Page 6, Offset 40262.
 -- ここでは光合成ができないのか、植物も呼吸をする為に酸素を大量消費するばかりで、大型の動物に分け与えられる酸素が無いのだ。
-Perhaps it's because photosynthesis isn't possible here, but since the plants are constantly consuming large amounts of oxygen to breathe, there's none left for other large animals.
+I don't know if photosynthesis simply isn't possible down here,
+ or if the plants just consume a lot of oxygen as part of respiration,
+ but there's none left to share with the larger animals.
 }
 [sha:ba2f15cb42f1cba7f00686a64d3864cf2840ea39]{
 -- Page 7, Offset 40263.
 -- 『上空の気流の乱れはこの酸素濃度が原因か。
-"So this concentration of oxygen must be the source of the disturbance in the atmosphere.
+%{i}
+"So this dearth of oxygen is responsible for the disturbance in the atmosphere above.
+%{/i}
 }
 [sha:8dc3266dca68b1078a3841397daea326e8c0fcf2]{
 -- Page 7, Offset 40264. Mods: @k, @e.
 -- 人類が知る原生植物ではあるが、
-These are the primordial plants known to humanity,
+%{i}
+These are prehistoric plants that humans have studied―――
+%{/i}
 }
 [sha:af4da133e30ee7195a8ebbc2934db1bab7432864]{
 -- Page 7, Offset 40265.
 -- その規格、強靭さはまるで別物だな』
-but its toughness make it an entirely different thing."
+%{i}
+But their strength makes them seem like a different species altogether."
+%{/i}
```

Comment:
hmm, long enough without capturing it well enough, and it should also link better
>But with their size and might, they may as well be something else altogether.